### ISSUE TITLE
no grapsh is show train warning

### ISSUE THREAD

**Issue by fortunelee** [deepfakes/faceswap#736]
Logs：
Loading...
05/29/2019 16:25:18 INFO     Log level set to: INFO
Using TensorFlow backend.
05/29/2019 16:25:21 INFO     Model A Directory: C:\Users\lipeng\faceswap\workspace\data_dst\aligned
05/29/2019 16:25:21 INFO     Model B Directory: C:\Users\lipeng\faceswap\workspace\data_src\aligned
05/29/2019 16:25:21 INFO     Training data directory: C:\Users\lipeng\faceswap\workspace\model
05/29/2019 16:25:21 INFO     ===============================================
05/29/2019 16:25:21 INFO     - Starting                                    -
05/29/2019 16:25:21 INFO     - Press 'ENTER' to save and quit              -
05/29/2019 16:25:21 INFO     - Press 'S' to save model weights immediately -
05/29/2019 16:25:21 INFO     ===============================================
05/29/2019 16:25:22 INFO     Loading data, this may take a while...
05/29/2019 16:25:22 INFO     Loading Model from Original plugin...
05/29/2019 16:25:25 INFO     Using configuration saved in state file
05/29/2019 16:25:25 WARNING  From E:\anaconda\envs\faceswap\lib\site-packages\tensorflow\python\framework\op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.\nInstructions for updating:\nColocations handled automatically by placer.
05/29/2019 16:25:36 INFO     Loaded model from disk: 'C:\Users\wblipeng1\faceswap\workspace\model'
05/29/2019 16:25:36 INFO     Loading Trainer from Original plugin...
05/29/2019 16:25:41 INFO     Enabled TensorBoard Logging
05/29/2019 16:25:41 WARNING  From E:\anaconda\envs\faceswap\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.\nInstructions for updating:\nUse tf.cast instead.
05/29/2019 16:27:55 INFO     saved models
05/29/2019 16:30:59 WARNING  No handles with labels found to put in legend.

Question：
train model , no grapsh is show ,click refresh button ,logs print No handles with labels found to put in legend.
